src/types/drupal/paragraph.ts

```diff
@@ -111,12 +115,30 @@ export interface ParagraphFeaturedContent extends DrupalParagraph {
   field_cta?: ParagraphButton
 }
 
+export type CCFieldCta = {
+  pid: string
+  target_id: string
+  type: string
+  field_button_label: Array<CCString>
+  field_button_link: Array<CCLink>
+}
+
 export interface ParagraphCCFeaturedContent {
+  target_id: string
   fetched: {
     // This normalizes the centralized content field_cta field to allow formatting
-    field_cta: Omit<ParagraphButton, 'drupal_internal__id' | 'id'>[]
+    field_cta: CCFieldCta[]
```

Comment:
This is potentially breaking for the VetCenter page. I checked, but maybe there is something I missed.